en/content/visiting.html

```diff
@@ -83,26 +83,16 @@ <h2 id="fields">Fields</h2>
 
 <h2 id="data-export-import">Data export / import</h2>
 <p>
-Export data to stdout:
-<pre>
-$ vespa-visit
-</pre>
+Export data to stdout.
 As a full data dump takes time / space / resources, refine as needed:
 <pre>
-$ vespa-visit -v -p progress-file -s 'id.hash().abs() % 100 = 0' \
-  --fieldset 'music:[document]' > 100.json
+$ vespa-visit -v -p progress-file -s 'id.hash().abs() % 100 = 0' > 100.json
```

Comment:
Should we replace this with a `/document/v1` example that uses slicing instead? The `id.hash()` construction requires a complete iteration, whereas slicing inherently limits to a subset of the data space. However, slicing is currently not supported by the `vespa-visit` CLI.

Reply:
That is a great suggestion! I punt this to the next PR, adding this as a ToDo, as I hope we can get the vespa-cli to support visiting and might rewrite examples at a larger scale.